### DISCUSSION TITLE
Não me aparecem os contratos na CHIA POOL

### DISCUSSION THREAD

**Discussion by FernandoCosta222** [Chia-Network/chia-blockchain#8531] [Support]
Preciso de ajuda. Tenho 2 computadores com o programa Chia. Uso a versão 1.2.7 nos dois.
No meu computador principal, que uso para plotar e farmar, quando abro o programa aparecem-me os botões "Nó Completo", "Plots", "Farm", "Pool" e "Chaves".
Os meus 700 plots estão cultivando (estado de cor verde). 
Quando carrego no botão "Pool", só me surge a opção de me juntar a uma Pool, mas os contratos que tinha não surgem. 
Quando escolho juntar-me a uma pool cobra o mojo mas não abre novo contrato.
As recompensas dos contratos na pool não estão a entrar na wallet.
Como conseguir voltar de novo a ter acesso aos contratos onde tenho os meus plots criados e voltar a receber as recompensas?

**Comment by loppefaaret** [comment]
you need to let the wallet sync pass the point of the last time you issued a change to the plotNFT - once it gets through syncing, it whould appear in the "Pool" tab... if you have copied over the wallet database, you need to remove it, and let the wallet resync from 0

**Comment by FernandoCosta222** [comment]
Thanks

Obter Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by BrandtH22** [comment]
@FernandoCosta222 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
